python/doc/user_manual/base_objects.rst

```diff
@@ -141,4 +141,6 @@ K-d tree
     :toctree: _generated/
     :template: class.rst_t
 
+    NearestNeighbourAlgorithm
+    RegularGridNearestNeighbour
```

Comment:
NaiveNearestNeighbour may be missing here

Reply:
Indeed.  This section must also be renamed, it is currently "K-d tree", but this is no more accurate with these changes.  And EnclosingSimplex classes will be added there too.  I am thinking about "Spatial lookup", any other idea?

Reply:
Nope, in Scipy it's "Spatial algorithms and data structures":
https://docs.scipy.org/doc/scipy-0.14.0/reference/spatial.html

Reply:
Spatial lookup sounds good.